lib/cli.rb

```diff
@@ -189,13 +189,24 @@ def burndown
     end
   end
 
-  desc 'plot SPRINT-NUMBER [--output] [--no-tasks] [--with-fast-lane]', 'Plot burndown chart for given sprint'
+  desc 'plot SPRINT-NUMBER [--output] [--no-tasks] [--with-fast-lane] [--plot-to-board]', 'Plot burndown chart for given sprint'
   option :output, aliases: :o, desc: 'Output directory', required: false
   option 'with-fast-lane', desc: 'Plot Fast Lane with new cards bars', required: false, type: :boolean
   option 'no-tasks', desc: 'Do not plot tasks line', required: false, type: :boolean
+  option :plot_to_board, type: :boolean, desc: 'Send the plotted data to the first card of Done column'
   def plot(sprint_number)
     process_global_options options
-    BurndownChart.plot(sprint_number, options)
+    if options[:plot_to_board]
+      begin
+        chart = BurndownChart.new @@settings
+        chart.update(options)
```

Comment:
This is also modifying the `yml` file, which we don't want to modify here. We only want to print the image and push it to the card.

Reply:
Please help identifying the line at which it modifying the .`yml` file. And by .`yml` file do you mean burndown-data-01.yml file which is created at time of initialization

Reply:
inside of the `update` method.

Yes, the `burndown-data-i.yaml` file, where i is the number of the sprint

Reply:
@Ana06 I dont see any change in the yml file. Please check now is it good to merge?